local-cli/runAndroid/adb.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+var child_process = require('child_process');
+
+/**
+ * Parses the output of the 'adb devices' command
+ */
+function parseDevicesResult(result) {
+  if(!result) return;
```

Comment:
Style, those apply to all of the code:
Always put `if` content on a new line with brackets
Space after the `if` keyword
Space before and after operators